etna/lib/etna/route.rb

```diff
@@ -64,19 +67,27 @@ def parts
     end
 
     def call(app, request)
-      start = Process.clock_gettime(Process::CLOCK_MONOTONIC)
+      update_params(request)
```

Comment:
Do this -before- the instrumentation logic so that the project_name param can be extracted.